packages/domain-search/src/hooks/use-suggestions-list.ts

```diff
@@ -1,23 +1,47 @@
-import { type DomainAvailability, DomainAvailabilityStatus } from '@automattic/api-core';
+import {
+	type DomainAvailability,
+	DomainAvailabilityStatus,
+	fetchDomainAvailability,
+} from '@automattic/api-core';
 import { useQueries, useQuery, UseQueryResult } from '@tanstack/react-query';
 import { useCallback, useMemo } from 'react';
 import { getTld } from '../helpers';
 import { partitionSuggestions } from '../helpers/partition-suggestions';
 import { useDomainSearch } from '../page/context';
 
 export const useSuggestionsList = () => {
-	const { query, queries, config } = useDomainSearch();
+	const { query, queries, config, events } = useDomainSearch();
 
 	const { data: suggestions = [], isLoading: isLoadingSuggestions } = useQuery( {
 		...queries.domainSuggestions( query ),
 		enabled: true,
+		// We should just get suggestions when the query changes
+		staleTime: Infinity,
+		refetchOnMount: false,
+		refetchOnWindowFocus: false,
```

Comment:
Why do we need that?

Reply:
Without that, the suggestions request is done every time the page  is focused (e.g. when you move from one tab to another)

Reply:
I see. Why is that a problem? Should we do that for the other queries as well?